packages/bsmi_ocaml/bsmi_ocaml.0.1.3/opam

```diff
@@ -0,0 +1,38 @@
+# This file is generated by dune, edit dune-project instead
+opam-version: "2.0"
+synopsis: "Bsmi ocaml library"
+description: "A very useful tool collections"
```

Comment:
Could you try being a little more specific? What is Bsmi?

Reply:
bsmi is just a name prefix, our OSS community has it's name , prefix with bsmi